bin/aptos

```diff
@@ -60,25 +72,35 @@ const main = async () => {
       path = "";
     }
   } else {
-    path = `${__dirname}/${binaryName}`;
+    path = `${__dirname}/${PNAME}`;
   }
 
   // If binary does not exist, download it.
   if (!fs.existsSync(path)) {
-    const url = `https://github.com/aptos-labs/aptos-core/releases/download/${PNAME}-v${VERSION}/${PNAME}-${VERSION}-${os}-x86_64.zip`;
+    // Look up the latest version.
+    let latestVersion;
+    if (os === "MacOS") {
+      latestVersion = getLatestVersionBrew();
+    } else {
+      latestVersion = await getLatestVersionGh();
+    }
+
+    console.log(`Downloading aptos CLI version ${latestVersion}`);
+
+    const url = `https://github.com/aptos-labs/aptos-core/releases/download/${PNAME}-v${latestVersion}/${PNAME}-${latestVersion}-${os}-x86_64.zip`;
```

Comment:
url is used only on windows/linux right?

Reply:
Yep, I could change the conditional logic so we have a "if windows or linx branch". But just making this const here is nbd.